internal/controller/init.go

```diff
@@ -1,17 +1,21 @@
 package controller
 
 import (
+	"fmt"
 	"github.com/kintoproj/kinto-cli/internal/config"
+	"github.com/kintoproj/kinto-cli/internal/utils"
 )
 
 //Set kintoCoreHost for CLI or reset it to default production host.
 func (c *Controller) Init(kintoCoreHost string) {
 
-	if kintoCoreHost == "Default" || kintoCoreHost == "default" {
+	if kintoCoreHost == "Reset" || kintoCoreHost == "reset" {
```

Comment:
action - make "reset" a constant.
And use https://golang.org/pkg/strings/#EqualFold